flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/VertexSlotSharingTest.java

```diff
@@ -90,12 +90,12 @@ public void testAssignSlotSharingGroup() {
 				new SerializedValue<>(new ExecutionConfig()),
 				AkkaUtils.getDefaultTimeout(),
 				new NoRestartStrategy(),
-				new Scheduler(TestingUtils.defaultExecutionContext()));
+				new TestingSlotProvider(ignored -> new CompletableFuture<>()));
 			eg.attachJobGraph(vertices);
 			
 			// verify that the vertices are all in the same slot sharing group
-			SlotSharingGroup group1 = null;
-			SlotSharingGroup group2 = null;
+			SlotSharingGroup group1;
```

Comment:
It is accidental to be in the same commit. I am not quite sure if I have to revert it. What is the most convenient commits layout for merging a pull request and how does flink deal with such "code cleaning" works? @zentol

Reply:
I would revert this as I wouldn't even categorize this as code-cleanup. Whether to keep these uninitialized or not is more or less personal preference.

Reply:
Gotcha.

Reply:
Revert while rebasing.